.github/workflows/main.yml

```diff
@@ -39,3 +39,9 @@ jobs:
         run: yarn test
         env:
           CI: true
+      - name: Test Install time
+        run: |
+          cd ../ && mkdir test && cd test
+          yarn cache clean
```

Comment:
We probably want to use a different yarn here as clearing the cache will slow our build.

Reply:
Makes sense. I hoped to just use some `--no-cache` or smth, but Yarn apparently doesn't have that flag ... What do you recommend?

Reply:
Can we get away with running this as a separate job?

Reply:
I think that would be ideal to run it as a separate job

Reply:
Done in c55ba95